server/src/ClientToDBAPI/UsersRoute/mainRoute.ts

```diff
@@ -171,6 +172,38 @@ usersRoute.post('/changeInvestigator', adminMiddleWare, (request: Request, respo
         });
 });
 
+usersRoute.get('/userTypes', (request: Request, response: Response) => {
+    logger.info({
+        service: Service.SERVER,
+        severity: Severity.LOW,
+        workflow: 'Getting user types',
+        step: 'querying the graphql API',
+        user: response.locals.user.id
+    });
+    graphqlRequest(GET_ALL_USER_TYPES, response.locals)
+        .then((result: GetAllUserTypesResponse) => {
+            if (result && result.data && result.data.allUserTypes) {
```

Comment:
You can use the ?. syntax

Reply:
In addition, you need to add log for the else clause and send response with error message

Reply:
fixed